src/CityData.js

```diff
@@ -41,7 +44,7 @@ export default class CityData extends Component {
     }
 
     getForecast = async () => {
-        const url = `${process.env.REACT_APP_SERVER_URL}/weather?city=${this.state.location.display_name}&lat=${this.state.location.lat}&lon=${this.state.location.lon}`;
+        const url = `${process.env.REACT_APP_SERVER_URL}weather?lat=${this.state.location.lat}&lon=${this.state.location.lon}`;
```

Comment:
Hi there!
Review url line 47. You are missing `/` after your `REACT_APP_SERVER_URL`